src/Microsoft.DotNet.ProjectModel/Project.cs

```diff
@@ -84,6 +84,8 @@ public string ProjectDirectory
 
         public bool IsTestProject => !string.IsNullOrEmpty(TestRunner);
         
+        public bool OverrideIsRunnable { get; set; }
```

Comment:
I get that, but I see this as a hack so we can force a non-synchronized project.json and csproj to behave in a certain way.
Why make a new property for this hack rather than reusing the property that is already there is what I'm getting at. 

If we see <OutputType>Exe</OutputType> why not set compilationOptions.EmitEntrypoint to true

---
In reply to: [71797149](https://github.com/dotnet/cli/pull/3911#discussion_r71797149) [](ancestors = 71797149,71797075)